src/views/Results.vue

```diff
@@ -139,6 +139,13 @@ export default {
 		},
 	},
 
+	watch: {
+		// Reload results, when form changes
+		'form.hash'() {
+			this.loadFormResults()
+		},
```

Comment:
`form.hash` is not supposed to ever change in the logic :)
`form` does change. `hash` too. ;)
This syntax is confusing :)

```suggestion
		hash() {
			this.loadFormResults()
		},
```

Reply:
You're right. Strange logic^^ 👍

Reply:
Ahh, but now it is not working anymore... 🤔

Reply:
What do you say about using `$route.hash`? That is in fact the intention behind..